app/translations/en.json

```
@@ -966,7 +966,22 @@
                 "package": "Upload a package",
                 "close": "Close",
                 "catalog": "Plugins Catalog"
-            }
+            },
+            "columns": {
+                "plugin": "Plugin",
+                "packageName": "Package name",
+                "packageVersion": "Package version",
+                "supportedPlatform": "Supported platform",
+                "distribution": "Distribution",
+                "distributeRelease": "Distribute release",
```

I think we can rename it to:
```suggestion
                "distributionRelease": "Distribution release",
```
as the actual name of the field used to show it is `distribution_release`.

---

Done